### ISSUE TITLE
显存占用很奇怪

### ISSUE THREAD

**Issue by lyz22233** [OpenRLHF/OpenRLHF#395]
RLHF全参数微调13B的模型，初始化ActorModel后，显存占用很奇怪。

机器设置：8个节点，每个节点1张卡；
actor_num_nodes==4，
actor_num_gpus_per_node==1
zero3
adam_offload

<img width="1517" alt="output" src="https://github.com/user-attachments/assets/faaa7863-39db-43b3-81cd-acb911ae7dd7">

初始化Actor后，每个节点的内存占用综合大概45G * 4，大概就是优化器状态参数的大小；
但是显存每张卡都占用了55G左右，这个很奇怪，是哪里设置的问题吗？

**Comment by hijkzzz** [comment]
有没有完整的配置?

**Comment by lyz22233** [comment]
> 有没有完整的配置设定?

ray job submit --address="http://127.0.0.1:8265" \
   --runtime-env-json='{"working_dir": "/home/work/liuyawen/code/llm_alignment_open_rlhf"}' \
   -- python3 -m openrlhf.cli.train_ppo_ray \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 1 \
   --reward_num_nodes 1 \
   --reward_num_gpus_per_node 1 \
   --critic_num_nodes 1 \
   --critic_num_gpus_per_node 1 \
   --actor_num_nodes 4 \
   --actor_num_gpus_per_node 1 \
   --vllm_num_engines 1 \
   --vllm_tensor_parallel_size 1 \
   --pretrain /home/work/models/compass_pro_v_1_2_0 \
   --reward_pretrain /home/work/models/compass_pro_v_1_2_0 \
   --save_path /home/work/models/ppo_train/compass_rlhf_001 \
   --micro_train_batch_size 2 \
   --train_batch_size 16 \
   --micro_rollout_batch_size 16 \
   --rollout_batch_size 32 \
   --max_samples 100000 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --generate_max_len 1024 \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 5e-7 \
   --critic_learning_rate 9e-6 \
   --init_kl_coef 0.01 \
   --prompt_data OpenRLHF/prompt-collection-v0.1 \
   --input_key context_messages \
   --apply_chat_template \
   --normalize_reward \
   --adam_offload \
   --gradient_checkpointing \
   --use_wandb wandb_token

**Comment by lyz22233** [comment]
> 有没有完整的配置设定?

不好意思，看起来是ray的dashboard中显示的有问题，我们用额外的监控就正常了。

**Comment by lyz22233** [comment]
~
